docs/reference/commandline/swarm_init.md

```diff
@@ -135,8 +135,10 @@ frequently.
 
 ## Related information
 
+* [node rm](node_rm.md)
```

Comment:
I agree that remove it. It was added for by thaJeztah  [Add "auto-accept=none" to documentation]. Now auto-accept was nonexistent in swarm init, so I think that remove the link.
@thaJeztah right?
Thanks